config/_shared.json

```diff
@@ -245,6 +245,7 @@
 		"ecommerce-monthly",
 		"ecommerce-2y",
 		"ecommerce-3y",
+		"entrepreneur",
```

Comment:
This is just to make sure the flow is using the new reskinned design as opposed to the old one with blue background.